tests/features/piwik.feature

```diff
@@ -1,22 +1,20 @@
-@wip
+@api
 Feature: Check Piwik
   In order to check if the the type attribute is set for the Piwik element.
```

Comment:
As you have a conflict to resolve on your branch, mayb you can take the opportinity to fix this small typo by removing the  useless additional "the" in  "check if the *the* type attribute"